src/managers/payment.ts

```diff
@@ -19,3 +21,13 @@ export const startPayment = (
 export const cancelPayment = (): Promise<void> => {
   return MobilePaymentsSdkReactNative.cancelPayment();
 };
+
+export namespace OfflinePaymentQueue {
+  export const getPayments = (): Promise<OffLinePayment[]> => {
+    return MobilePaymentsSdkReactNative.getPayments();
```

Comment:
Nit: if this is scoped as `MobilePaymentsSdkReactNative` we might want the naming to be `.getOfflinePayments`?  I don't think we'll have a `getOnlinePayments` ever, but if we do, this naming would collide?